src/sagemaker_inference/utils.py

```diff
@@ -66,3 +66,16 @@ def retrieve_content_type_header(request_property):
             return request_property[key]
 
     return None
+
+
+def parse_accept(accept):
+    """Parses the Accept header sent with a request.
+
+    Args:
+        accept (str): the value of an Accept header.
+
+    Returns:
+        (list): A list containing the MIME types that the client is able to
+            understand.
+    """
+    return accept.split(", ")
```

Comment:
is it possible it might be comma-delimited without spaces?

Reply:
I couldn't find anything about that in the specification https://www.w3.org/Protocols/rfc2616/rfc2616-sec14.html, but in all of the examples they use commas with spaces. Also, we would be encoding the Accept header with the Python SDK for most use cases (I think), in which case we would use spaces.

Reply:
I went and looked at a bunch of Stack Overflow posts and Mozilla's documentation, and it does seem like most people use a space. However, https://developer.mozilla.org/en-US/docs/Glossary/quality_values (linked from [this page](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Accept)) seems to imply that the space isn't always needed 🤷‍♀️